packages/widget-v2/src/modals/TokenAndChainSelectorModal/TokenAndChainSelectorModal.tsx

```diff
@@ -157,12 +157,7 @@ export const TokenAndChainSelectorModal = createModal(
     const filteredAssets = useMemo(() => {
       if (!groupedAssetsByRecommendedSymbol) return;
       return matchSorter(groupedAssetsByRecommendedSymbol, searchQuery, {
-        keys: [
-          "id",
-          "assets.*.symbol",
-          "assets.*.denom",
```

Comment:
we can keep the denom

Reply:
@codingki the denom seems to match groups that the user probabbly wouldn't expect, for example with osm (the example in linear)

![image](https://github.com/user-attachments/assets/2ef791b6-204b-446a-a15b-3de7b7056314)